xcode/Cinder-SdfText.xcodeproj/project.pbxproj

```diff
@@ -1865,6 +1889,7 @@
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				HEADER_SEARCH_PATHS = (
 					"$(CINDER_PATH)/include",
+					"$(CINDER_PATH)/include/freetype",
```

Comment:
I wasn't sure what the best way to handle this was. We could just change `$(CINDER_PATH)/include` to be recursive `$(CINDER_PATH)/include/**` but since we only needed on directory I opted to be explicit.